code/datums/materials/_material.dm

```diff
@@ -0,0 +1,67 @@
+/*! Material datum
+
+Simple datum which is instanced once per type and is used for every object of said material. It has a variety of variables that define behavior. Subtyping from this makes it easier to create your own materials. 
+
+*/
+
+
+/datum/material
+	var/name = "material"
+	var/desc = "its..stuff."
+	//Var that's mostly used by science machines to identify specific materials, should most likely be phased out at some point
+	var/id = "mat"
+	///Base color of the material, is used for greyscale. Item isn't changed in color if this is null.
+	var/color
+	///Base alpha of the material, is used for greyscale icons.
+	var/alpha
+	///Materials "Traits". its a map of key = category | Value = Bool. Used to define what it can be used for.gold
+	var/list/categories = list()
+	///The type of sheet this material creates. This should be replaced as soon as possible by greyscale sheets.
+	var/sheet_type
+	///The type of coin this material spawns. This should be replaced as soon as possible by greyscale coins.
+	var/coin_type
+	///This is a modifier for force, and resembles the strength of the material
+	var/strength_modifier = 1
+	///This is a modifier for integrity, and resembles the strength of the material
+	var/integrity_modifier = 1
+
+///This proc is called when the material is added to an object.
+/datum/material/proc/on_applied(atom/source, amount, material_flags)
+	if(!(material_flags & MATERIAL_NO_COLOR)) //Prevent changing things with pre-set colors, to keep colored toolboxes their looks for example
```

Comment:
is this a workaround the fact that not everything uses the atom colours system?

Reply:
its mainly because we have toolboxes that start colored. I'm going to work around this in a different way once I rework paintable items hopefully